browser/resources/settings/brave_overrides/privacy_page.ts

```diff
@@ -52,10 +52,10 @@ function InsertGoogleSignInSubpage (
       } else {
         googleSignInDefault.setAttribute(
           'toggle-off-label',
-          I18nBehavior.i18n('siteSettingsGoogleSignInBlock'))
+          I18nBehavior.i18n('siteSettingsGoogleSignInAsk'))
```

Comment:
This is just a side note (and TODO to fix later, cc @emerick as it looks like you may have edited this file most often recently) as it's the first time I've really looked in to this override file recently, but it's a little strange that we're creating some html elements via `getTrustedHtml` and then using `querySelector` to go through it and change attributes. The `settings-privacy-page` component _does_ have the `I18nMixin` so I assume we can just write the i18n string names inside the html with polymer bindings, e.g. `toggle-on-label="$i18n{siteSettingsGoogleSignInAsk}"`